feconf.py

```diff
@@ -861,3 +861,47 @@ def get_empty_ratings():
 QUESTION_BATCH_SIZE = 10
 
 STATE_ANSWER_STATS_MIN_FREQUENCY = 2
+
+# RTE content specifications according to the type.
```

Comment:
One last-minute thing: this comment is not very clear. Please modify the comment to clarify what the "type" you are referring to here is.

I know I already LGTM so if this reaches you too late make the change in 4985.

Reply:
Fixed!